config.go

```diff
@@ -98,6 +108,16 @@ type Config struct {
 	// directory defined by StaticRoot.
 	ServeStatic bool `long:"servestatic" description:"Flag to enable or disable static content serving."`
 
+	// DatabaseBackend is the database backend to be used by the server.
+	DatabaseBackend string `long:"databasebackend" description:"The database backend to use for storing all asset related data." choice:"sqlite" choice:"postgres"`
```

Comment:
How about `db_backend` for the `long` here as it's shorter?

Reply:
none of the other config params separates its words. I like `db_backend` more but I think we should use `dbbackend` to be consistent